lib/db/mysql.js

```diff
@@ -571,6 +574,26 @@ module.exports = function (log, error) {
       })
   }
 
+  // Delete : unverifiedTokens
+  // Where  : tokenVerificationCode = $1, uid = $2
+  var VERIFY_TOKEN_CODE = 'CALL verifyTokenCode_1(?, ?)'
+
+  MySql.prototype.verifyTokenCode = function (tokenVerificationCode, accountData) {
+    return this.readFirstResult(VERIFY_TOKEN_CODE, [createHash(tokenVerificationCode), accountData.uid])
+      .then(function (result) {
+        if (result['@rowCount'] === 0) {
```

Comment:
We don't use this `@rowCount` check anywhere else, is this the only way we can find out if nothing was found?

Reply:
Unfortunately, this was the only way I could find that would return the row count from within a transaction. That might also explain why the `affectedRows` property was not returned.